backend/oasst_backend/utils/exported_tree_loading.py

```diff
@@ -0,0 +1,75 @@
+import json
+from collections import defaultdict
+from typing import List
+
+import pandas as pd
+
+
+def load_jsonl(filepaths):
+    data = []
+    for filepath in filepaths:
+        with open(filepath, "r") as f:
```

Comment:
not sure if you saw it, we have fully typed pydantic classes for loading message trees, e.g. see use in import.py: https://github.com/LAION-AI/Open-Assistant/blob/e963ca3b868e57f7d7d5ab8332bc5a8d59d1d99c/backend/import.py#L115-L118

Reply:
did not see that. looking through it now :)